src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,52 @@
+package lotto.view;
+
+import java.util.stream.Collectors;
+import lotto.controller.dto.LottoDrawResult;
+import lotto.controller.dto.LottoDto;
+import lotto.controller.dto.LottosDto;
+
+public final class OutputView {
+
+    public static void printPurchasedLottos(final LottosDto lottosDto) {
+        System.out.printf("\n%d개를 구매했습니다.\n", lottosDto.lottoDtos().size());
+        final String rendered = renderLottosDto(lottosDto);
+        System.out.println(rendered);
+    }
+
+    public static void printDrawingResult(final LottoDrawResult result) {
+        System.out.printf("""
+                                        
+                        당첨 통계
+                        ---
+                        3개 일치 (5,000원) - %d개
+                        4개 일치 (50,000원) - %d개
+                        5개 일치 (1,500,000원) - %d개
+                        5개 일치, 보너스 볼 일치 (30,000,000원) - %d개
+                        6개 일치 (2,000,000,000원) - %d개
+                        총 수익률은 %.1f%%입니다.
+                        """.stripIndent(),
+                result.fifth(),
+                result.fourth(),
+                result.third(),
+                result.second(),
+                result.first(),
+                result.totalEarningReturn()
+        );
+    }
+
+    private static String renderLottosDto(final LottosDto lottosDto) {
+        return lottosDto.lottoDtos()
+                .stream()
+                .map(OutputView::renderLottoDto)
+                .collect(Collectors.joining("\n"));
```

Comment:
코드 리뷰를 하던 중 상황에 따라 개행을 하기 위해 `\n`이 아닌 다른 문자가 사용될 수도 있다는 언급을 했던 것이 기억이 납니다.
물론 정말 사소한 부분이겠지만, 혹시라도 시간이 남아 리팩토링을 신경 쓸 수 있다면 이런 부분도 신경 써 보면 좋지 않을까 하는 생각에 코멘트 남깁니다!

```suggestion
                .collect(Collectors.joining(System.lineSeparator()));
```

Reply:
말씀하신대로 UNIX 계열에서는 개행 문자가 `"\n"`로, 윈도우에서는 `"\r\n"`로 플랫폼마다 다르게 표기되는 부분이 문제가 될 수 있겠네요.

기계적으로 작성하느라 생각치 못했는데 꼼꼼하게 잡아주셔서 감사합니다.😊

Reply:
@zangsu 
저는 `System.out.println()` 을 메서드로 만들어 사용했는데 말씀 주신 부분이 훨씬 편해 보이네요. 

이런 섬세한 부분은 혁수 님의 굉장한 강점이라고 생각해요. 좋은 정보 감사합니다. 😄😄